<<<<<<< HEAD
package _16_Exception_Handling;

public class Basic_try_catch {
    public static void main(String[] args) {

        try {
            int result = 10 / 0;
        } catch (ArithmeticException e) {
            System.out.println("Cannot divide by zero.");
        }
        System.out.println("running");
    }
}
=======
package _16_Exception_Handling;

public class Basic_try_catch {
    public static void main(String[] args) {

        try {
            int result = 10 / 0;
        } catch (ArithmeticException e) {
            System.out.println("Cannot divide by zero.");
        }
        System.out.println("running");
    }
}
>>>>>>> 35387576b9a0b809457b398be6486ca5e6dbc9f1
